shared-mime-info: update to 1.14.

mobinmob opened the pull request.

Hoshpak commented:
This way autogen.sh will execute configure without our configure_args before the actual configure step of the build process. Usually, running autogen.sh with `NOCONFIGURE=1` set will prevent that. The autogen script also misses autopoint when cross-compiling which is part of gettext-devel so it seems this package is needed in hostmakedepends to cross-build.

mobinmob commented:
> This way autogen.sh will execute configure without our configure_args before the actual configure step of the build process. Usually, running autogen.sh with `NOCONFIGURE=1` set will prevent that. The autogen script also misses autopoint when cross-compiling which is part of gettext-devel so it seems this package is needed in hostmakedepends to cross-build.

I moved getttext-devel from makedepends to hostmakedepends and added NOCONFIGURE=1. Thank you :)

Hoshpak commented:
Actually it might be needed in both. In hostmakedepends to have the binary available on the host and in makedepends to have the headers available to compile. We will see what travis says.
